### PR TITLE
Add documentation for OTel runtime metrics

### PR DESCRIPTION
**Pull request by liustanley**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by kayayarai** [comment]
Opened [DOCS-5658](https://datadoghq.atlassian.net/browse/DOCS-5658) to track editorial review

File: content/en/opentelemetry/runtime_metrics/java.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+|	`jvm.gc.minor_collection_time` | The fraction of time spent in minor garbage collection. Set `new_gc_metrics: true` to receive this metric. | N/A |
+|	`jvm.os.open_file_descriptors` | | N/A |
+
+
+[1]: /opentelemetry/otel_metrics
+[2]: https://app.datadoghq.com/integrations/java
+[3]: https://opentelemetry.io/docs/instrumentation/java/automatic/
+[4]: https://opentelemetry.io/docs/instrumentation/java/manual/
+[5]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/runtime-metrics-java8/library
+[6]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/runtime-metrics-java17/library
```
Comment (kayayarai) [inline content/en/opentelemetry/runtime_metrics/java.md:64]: Both of these links 404. Are they the wrong URLs, or will there be something there eventually?

File: content/en/opentelemetry/runtime_metrics/go.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| `runtime.go.num_goroutine` | Number of goroutines spawned. | `process.runtime.go.goroutines` |
+| `runtime.go.num_cgo_call` | Number of CGO calls made. |`process.runtime.go.cgo.calls` |
+| `runtime.go.mem_stats.lookups` | Number of pointer lookups performed by the | `process.runtime.go.mem.lookups` |
+| `runtime.go.mem_stats.heap_alloc` | Bytes of allocated heap objects. | `process.runtime.go.mem.heap_alloc` |
+| `runtime.go.mem_stats.heap_sys` | Bytes of heap memory obtained from the operating system. | `process.runtime.go.mem.heap_sys` |
+| `runtime.go.mem_stats.heap_idle` | Bytes in idle (unused) spans. | `process.runtime.go.mem.heap_idle` |
+| `runtime.go.mem_stats.heap_inuse` | Bytes in in-use spans. | `process.runtime.go.mem.heap_inuse` |
+| `runtime.go.mem_stats.heap_released` | Bytes of physical memory returned to the operating system. | `process.runtime.go.mem.heap_released` |
+| `runtime.go.mem_stats.heap_objects` | Number of allocated heap objects. | `process.runtime.go.mem.heap_objects` |
+| `runtime.go.mem_stats.pause_total_ns` | Cumulative nanoseconds in GC. | `process.runtime.go.gc.pause_total_ns` |
```
Comment (kayayarai) [inline content/en/opentelemetry/runtime_metrics/go.md:33]: What's "GC"?

File: content/en/opentelemetry/runtime_metrics/go.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+| `runtime.go.mem_stats.total_alloc` | Cumulative bytes allocated for heap objects. | N/A |
+| `runtime.go.mem_stats.sys` | Total bytes of memory obtained from the operating system. | N/A |
+| `runtime.go.mem_stats.mallocs` | Cumulative count of heap objects allocated. | N/A |
+| `runtime.go.mem_stats.frees` | Cumulative count of heap objects freed. | N/A |
+| `runtime.go.mem_stats.stack_inuse` | Bytes in stack spans. | N/A |
+| `runtime.go.mem_stats.stack_sys` | Bytes of stack memory obtained from the operating system. | N/A |
+| `runtime.go.mem_stats.m_span_inuse` | Bytes of allocated mspan structures. | N/A |
+| `runtime.go.mem_stats.m_span_sys` | Bytes of memory obtained from the operating system for mspan structures. | N/A |
+| `runtime.go.mem_stats.m_cache_inuse` | Bytes of allocated mcache structures. | N/A |
+| `runtime.go.mem_stats.m_cache_sys` | Bytes of memory obtained from the operating system for | N/A |
```
Comment (kayayarai) [inline content/en/opentelemetry/runtime_metrics/go.md:46]: unexpected end of description

File: content/en/opentelemetry/runtime_metrics/go.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Runtime metric mappings
+
+The following table lists the Datadog runtime metrics that are supported by mapping OpenTelemetry runtime metrics, with "N/A" indicating that there is no OpenTelemetry counterpart.
+
+| Datadog metric | Description |  OpenTelemetry counterpart |
+| --- | --- | --- |
+| `runtime.go.num_goroutine` | Number of goroutines spawned. | `process.runtime.go.goroutines` |
+| `runtime.go.num_cgo_call` | Number of CGO calls made. |`process.runtime.go.cgo.calls` |
+| `runtime.go.mem_stats.lookups` | Number of pointer lookups performed by the | `process.runtime.go.mem.lookups` |
```
Comment (kayayarai) [inline content/en/opentelemetry/runtime_metrics/go.md:26]: unexpected end of description

**Review by kayayarai — CHANGES_REQUESTED** [review]
Hi! This is a wonderful PR! I gave it a bit of a makeover, please check for mistakes. Also left you some questions about some missing info. Going to as the PM for a screen shot of the default dashboards

File: content/en/opentelemetry/runtime_metrics/go.md (hunk 2 above)
Comment (liustanley) [reply content/en/opentelemetry/runtime_metrics/go.md:33]: GC stands for garbage collection

File: content/en/opentelemetry/runtime_metrics/java.md (hunk 1 above)
Comment (liustanley) [reply content/en/opentelemetry/runtime_metrics/java.md]: Thanks for catching this, the URLs were out of date so I just updated them

**Comment by liustanley** [comment]
> Hi! This is a wonderful PR! I gave it a bit of a makeover, please check for mistakes. Also left you some questions about some missing info. Going to as the PM for a screen shot of the default dashboards

@kayayarai I just reviewed your edits and everything looks good to me

File: content/en/opentelemetry/runtime_metrics/dotnet.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+| `runtime.dotnet.gc.count.gen1` | The number of gen 1 garbage collections. | `process.runtime.dotnet.`<br/>`gc.collections.count` |
+| `runtime.dotnet.gc.count.gen2` | The number of gen 2 garbage collections. | `process.runtime.dotnet.`<br/>`gc.collections.count` |
+| `runtime.dotnet.cpu.system` | The number of milliseconds executing in the kernel. | N/A |
+| `runtime.dotnet.cpu.user` | The number of milliseconds executing outside the kernel. | N/A |
+| `runtime.dotnet.cpu.percent` | The percentage of total CPU used by the application. | N/A |
+| `runtime.dotnet.mem.committed` | Memory usage. | N/A |
+| `runtime.dotnet.threads.count` | The number of threads. | N/A |
+| `runtime.dotnet.threads.workers_count` | The number of workers in the threadpool. (.NET Core only) | N/A |
+| `runtime.dotnet.threads.contention_time` | The cumulated time spent by threads waiting on a lock. (.NET Core only) | N/A |
+| `runtime.dotnet.gc.memory_load` | The percentage of the total memory used by the process. The GC changes its behavior when this value gets above 85. (.NET Core only) | N/A |
```
Comment (kayayarai) [inline content/en/opentelemetry/runtime_metrics/dotnet.md:42]:
```suggestion
| `runtime.dotnet.gc.memory_load` | The percentage of the total memory used by the process. The garbage collection (GC) changes its behavior when this value gets above 85. (.NET Core only) | N/A |
```

**Review by kayayarai — APPROVED** [review]
Good to go when the code ships :D

**Comment by buraizu** [comment]
Confirmed that code freeze is not in effect for documentation repo

**Pull request merged**